playground-remote-inference.md

```diff
@@ -0,0 +1,72 @@
+# Remote inference in Playground
+
+Users can chat with external language model in playground.
+
+## Prerequisites
+
+- AI Toolkit v0.4.0 or newer.
+- OpenAI compatible chat completion endpoint.
+
+## How to add a remote model
+
+1. Click the "+" icon when hovering on "MODELS" tree view or enter `AI Toolkit: Add model for remote inference` in command palette.
+
+    ![](./Images/playground-remote-inference/0-entrypoint-treeview.png)
+
+    ![](./Images/playground-remote-inference/1-entrypoint-command.png)
+
+2. Enter the model name and press Enter.
```

Comment:
In this text we need to tell that the name is user created name, not any existing names, etc.

Reply:
In the note, it mentions the name will be sent as the `model` parameter for the chat completion API.

Reply:
It is not only for display purpose.

Reply:
Updated

Reply:
so can it be any name user wants? The part try to make it clear is whether user need to follow something for giving the model name, or it is totally up to the user. They first time I tried this, I also not quite sure if there is any requirement of the name.

Reply:
Updated